### PR TITLE
Docker compose'ifying eyes-in-the-sky

### PR DESCRIPTION
**Pull request by LoungeFlyZ**
This PR includes a number changes to make eyes-in-the-sky docker-compose friendly:

1. move all container configuration to env vars
2. add suitable docker-compose.yml file
3. move all configuration to the docker-compose.yml file
4. updated readme with instructions on using docker-compose on the raspberry pi

Note: The compose currently points to my hub image.  This could be changed once the images from the original build were updated to use this new environment based config.

Thanks
-Chris

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,71 @@
...
+2. Clone the repo on your raspberry pi:
+
+    ```
+    $ git clone https://github.com/LoungeFlyZ/eyes-in-the-sky
+    ```
+
+3. Download a copy of docker-compose for raspberry pi:
+
+    ```
+    $ wget -O docker-compose https://github.com/javabean/arm-compose/releases/download/1.16.1/docker-compose-Linux-armv7l
```
Comment (alexellis) [inline README.md]: Get this via official route through pip

**Review by alexellis — COMMENTED** [review]
The MAC address and IP changes with each container and that data is used to track the unit - for this reason you need to set it up front.

File: README.md (hunk 1 above)
Comment (LoungeFlyZ) [reply README.md]: FlightAware moved to not using MAC address anymore.  They have a "feeder id" (in the compose yml file) now. Once up and running you can grab the feeder id you have been assigned and update your compose file.

Regarding compose installation.  I couldnt find how to get a more recent compose build for arm via pip.  any pointers?

**Comment by LoungeFlyZ** [comment]
FlightAware moved to not using MAC address anymore.  They have a "feeder id" (in the compose yml file) now. Once up and running you can grab the feeder id you have been assigned and update your compose file.

**Comment by LoungeFlyZ** [comment]
@alexellis i updated the readme on how to set the feeder id in the compose file to ensure it is not seen as a new feeder each time. I don't know how long ago them move to the id vs. the MAC address ... but it confused me for some time and i kept getting new feeders showing up in flightaware until i did some digging and found this was the case.

**Comment by alexellis** [comment]
Hey @LoungeFlyZ I'm OK with using `docker run` vs compose for everything - in this use-case it creates more problems than it's worth in my opinion. If you have fixes to updates to the Dockerfiles due to breaking changes etc then I'd be happy to take those.

Feel free to fork the repository and maintain your own compose files.

Thanks for the interest,

Alex

**Comment by LoungeFlyZ** [comment]
Thanks Alex,  yes i have a fork. I added a flight radar 24 feeder to it recently too.   Thanks for starting this!

**Comment by alexellis** [comment]
Please keep in touch via Twitter @alexellisuk or GitHub. Good luck

**Pull request closed without merging**